testcases/OpTestKernelTest.py

```diff
@@ -125,6 +125,65 @@ def Store_loc(self, er) :
             matches = [match.group(1) for match in re.finditer(pattern,er)]
             return matches
     
+    def boot_kernel(self):
+        """
+        Does kexec boot for Upstream Linux
+        """
+        base_version = self.con.run_command("uname -r")
+        ker_ver = self.con.run_command("make kernelrelease")[-1]
```

Comment:
the above steps works only after make olddefconfig step.. so move after that

Reply:
moved